codenames-front-end/src/components/game/stopped/menu/CNPlayer/CNPlayer.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {FC} from 'react';
+
+import css from "./CNPlayer.module.css"
+import {IUser} from "../../../../../models/CodeNames/IUser";
+
+interface CNPlayerProps{
+    user: IUser;
+    type?: "div" | "li"
+    className?: string;
+}
+
+const CNPlayer: FC<CNPlayerProps> = ({user, type, className}) => {
```

Comment:
Do not use FC (https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13)

Reply:
Reworked